packages/playground-preview-worker/src/index.ts

```diff
@@ -98,21 +98,21 @@ export class PreviewError extends HttpError {
 class TokenUpdateFailed extends HttpError {
 	name = "TokenUpdateFailed";
 	constructor() {
-		super("Provide token", 400, false);
+		super("Provide valid token", 400, false);
 	}
 }
 
 class RawHttpFailed extends HttpError {
 	name = "RawHttpFailed";
 	constructor() {
-		super("Provide token", 400, false);
+		super("Provide valid token", 400, false);
```

Comment:
Is it worth specifying the header name that is expected? E.g.

```suggestion
		super("Provide valid `X-CF-Token` header", 400, false);
```